githooks/pre-push

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+EXIT_STATUS=0
```

Comment:
Given that essentially all our repos have the same make targets, how about we move this to conduit commons, so you can then configure the same hook in all repos?